docker-images/kafka/scripts/kafka_config_generator.sh

```diff
@@ -43,7 +43,7 @@ if [ "$KAFKA_AUTHORIZATION_TYPE" = "simple" ]; then
   # Prepare super.users field
   KAFKA_NAME=$(hostname | rev | cut -d "-" -f2- | rev)
   ASSEMBLY_NAME=$(echo "${KAFKA_NAME}" | rev | cut -d "-" -f2- | rev)
-  SUPER_USERS="super.users=User:CN=${ASSEMBLY_NAME}-topic-operator,O=io.strimzi;User:CN=${KAFKA_NAME},O=io.strimzi"
+  SUPER_USERS="super.users=User:CN=${ASSEMBLY_NAME}-topic-operator,O=io.strimzi;User:CN=${KAFKA_NAME},O=io.strimzi;User:CN=${ASSEMBLY_NAME}-entity-operator,O=io.strimzi"
```

Comment:
Since the TO should be now using EO keys ... can we delete this: `User:CN=${ASSEMBLY_NAME}-topic-operator,O=io.strimzi`